tracer/src/Datadog.Trace/Iast/IastModule.cs

```diff
@@ -865,6 +866,25 @@ internal static void OnEmailHtmlInjection(object? message)
         GetScope(messageDuck.Body, IntegrationId.EmailHtmlInjection, VulnerabilityTypeName.EmailHtmlInjection, OperationNameEmailHtmlInjection, taintValidator: Always, safeSources: _dbSources, exclusionSecureMarks: SecureMarks.Xss);
     }
 
+    public static void LogAspectException(Exception ex, string aspectInfo)
+    {
+        try
+        {
+            if (!LoggedAspectExceptionMessages.Add(aspectInfo.GetHashCode()))
+            {
+                Log.Debug(ex, "Error invoking {AspectInfo}", aspectInfo);
```

Comment:
Should this be an error? Otherwise it's the same as the other branch?
```suggestion
                Log.Error(ex, "Error invoking {AspectInfo}", aspectInfo);
```

You also have a slight issue here in that it means you'll get _no_ useful information in telemetry, as the aspect info isn't included 🙁  

EDIT: Included a fix in the previous comment 🙂